client/src/sass/theme.sass

```diff
@@ -1,28 +1,22 @@
-$dk-text: #212529
-$text: #495057
-$lt-text: #6f7780
-$link: #1669bb
-$hover: #1a5a96
-$p-blue-text: #38598a
-
-$grey-1: #f1f3f5
+$grey-1: #f1f3f5 // same as $gray1
```

Comment:
These variables are still used by components that use SASS.

Eventually these must die...